DESCRIPTION

```diff
@@ -12,7 +12,8 @@ Authors@R: c(
     person("Ryan", "Thompson", email="rct@thompsonclan.org", role="aut"),
     person("Nitesh", "Turaga", role="aut"),
     person("Aaron", "Lun", role = "ctb"),
-    person("Henrik", "Bengtsson", role = "ctb"))
+    person("Henrik", "Bengtsson", role = "ctb")
+    person("Phylis", "Atieno", role = "ctb"))
```

Comment:
There is a comment missing from the preceding line. I think you should add a comment to your name to indicate what you have contributed
```
    person("Henrik", "Bengtsson", role = "ctb"),
    person("Phylis", "Atieno",
        role = "ctb",
        comment ="Translated 'Introduction to BiocParallel' vignette from Sweave to Rmarkdown / HTML."
    ))